aws-sdk-core/lib/aws-sdk-core/api/docs/param_formatter.rb

```diff
@@ -44,7 +44,7 @@ def ref_value(ref, i, visited)
             end
           when ListShape then list(ref, i, visited)
           when MapShape then map(ref, i, visited)
-          when BooleanShape then "true"
+          when BooleanShape then "false"
```

Comment:
Yes, that is only for the documented request syntax examples. Another option would be to do something like `"true || false"`. I would prefer to keep it valid Ruby syntax. The important part of this is to ensure the example shows that a boolean is provided.